code/modules/vehicles/mecha/equipment/weapons/mecha_ammo.dm

```diff
@@ -20,7 +22,7 @@
 	desc = rounds ? initial(desc) : "An exosuit ammuniton box that has since been emptied. It can be safely folded for recycling."
 
 /obj/item/mecha_ammo/update_icon_state()
-	icon_state = rounds ? initial(icon_state) : "empty"
+	icon_state = rounds ? initial(icon_state) : initial(icon_state) + "_e"
```

Comment:
"[initial(icon_state)]_e" might be more readable here

Reply:
```suggestion
	icon_state = rounds ? initial(icon_state) : "[initial(icon_state)]_e"
```